assets/style.css

```diff
@@ -19,6 +19,8 @@
 }
 
 body {
+	color: #222;
```

Comment:
Why #222 instead of the current #000? I thought higher contrast is better for accessibility reasons.

Reply:
I don't know. I've just got it from https://www.debian.org/debian.css body's element.

Reply:
Okay. I think in this case high contrast is better than consistency with other debian css files. Can you change it to #000 please?

Reply:
I've updated it to '#000' and now it's `Outdated` here. Could you help me please?

Reply:
The UI showing outdated is okay.